### DISCUSSION TITLE
Syncfusion

### DISCUSSION THREAD

**Discussion by DirtyWidget** [Gh0st352/DNA-Keycards-eEditor#11] [Q&A] [answered]
Is Syncfusion a requirement to Build this in VS?

**Comment by Gh0st352** [comment, marked as answer]
> Is Syncfusion a requirement to Build this in VS?

Yes. Syncfusion offers free licenses for devs that create free opensource tools if you are interested in applying for one.

(Although I suppose you could use this as a baseline to create a non-syncfusion version)
